<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ханойские башни</string>
  </property>
  <widget class="QWidget" name="container">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Нажмите, чтобы запустить программу</string>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>455</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>300</number>
    </property>
    <property name="value">
     <number>50000</number>
    </property>
    <property name="sliderPosition">
     <number>50000</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeed">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeedView">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeedMeasure">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>х</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPause">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Нажмите, чтобы приостановить программу</string>
    </property>
    <property name="text">
     <string>Пауза</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Нажмите, чтобы остановить программу</string>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonResume">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Нажмите, чтобы продолжить выполнение программы</string>
    </property>
    <property name="text">
     <string>Продолжить</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Очень быстро</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Медленно</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>480</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Быстро</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonShuffle">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Нажмите, чтобы перемешать кольца</string>
    </property>
    <property name="text">
     <string>Перемешать</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSort">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Нажмите, чтобы упорядочить кольца</string>
    </property>
    <property name="text">
     <string>Упорядочить</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>buttonStop</zorder>
   <zorder>buttonResume</zorder>
   <zorder>graphicsView</zorder>
   <zorder>speedSlider</zorder>
   <zorder>labelSpeed</zorder>
   <zorder>labelSpeedView</zorder>
   <zorder>labelSpeedMeasure</zorder>
   <zorder>buttonPause</zorder>
   <zorder>buttonRun</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>buttonShuffle</zorder>
   <zorder>buttonSort</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>&amp;Профиль</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="actionProgramme"/>
    <addaction name="actionAuthor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProfile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Загрузить профиль</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Сохранить профиль</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>&amp;Изменить...</string>
   </property>
  </action>
  <action name="actionProgramme">
   <property name="text">
    <string>&amp;О программе</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Об &amp;авторе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
